pydatastructs/linear_data_structures/tests/test_algorithms.py

```diff
@@ -90,3 +90,17 @@ def test_matrix_multiply_parallel():
     J = [[2, 1, 2], [1, 2, 1], [2, 2, 2]]
     output = matrix_multiply_parallel(I, J, num_threads=1)
     assert expected_result == output
+
+def test_bucket_sort():
+    random.seed(1000)
+
+    n = random.randint(10, 20)
+    arr = DynamicOneDimensionalArray(int, 0)
+    for _ in range(n):
+        arr.append(random.randint(1, 1000))
+    for _ in range(n // 3):
+        arr.delete(random.randint(0, n // 2))
+
+    expected_arr = [102, 134, 228, 247, 362, 373, 448,
+                    480, 548, 686, 688, 696, 779]
+    assert bucket_sort(arr)._data == expected_arr
```

Comment:
```suggestion
```

Reply:
Instead of the above add one line below the following, 

https://github.com/codezonediitj/pydatastructs/blob/87c7a80f2fcf0ce34c8df70f137fcb417a3c268c/pydatastructs/linear_data_structures/tests/test_algorithms.py#L53-L54

```python
def test_bucket_sort():
    _test_common_sort(bucket_sort)
```